bundles/python/python_lexer.moon

```diff
@@ -100,4 +132,24 @@ howl.aux.lpeg_lexer ->
       identifier,
       decorator
     }
+
+    tsq_interpolation: gen_string_interpolation 'ts'
```

Comment:
It would be nice to not have all variants of the `f` strings here. Instead just call these, say,  `f_string` as a single rule that is defined separately above using `any`. Also, style wise, it seems strange to use interpolation with 'ts' etc. to refer to other rules - it makes it somewhat hard to follow.  Would be possible to just generate all the variants of the f strings by iterating over the names of the normal strings? Maybe you could add the generated rules to a list and then make an `any` rule out of it?